rust/src/service/data.rs

```diff
@@ -182,3 +219,69 @@ impl Sum2Data {
         Ok(None)
     }
 }
+
+#[derive(Debug, PartialEq, Default)]
+pub struct RoundParametersData {
+    /// The coordinator public key for encryption.
+    pub pk: Option<CoordinatorPublicKey>,
+
+    /// Fraction of participants to be selected for the sum task.
+    pub sum: Option<f64>,
+
+    /// Fraction of participants to be selected for the update task.
+    pub update: Option<f64>,
+
+    /// The random round seed.
+    pub seed: Option<RoundSeed>,
+
+    /// The global model of the previous round.
+    pub global_model: Option<SerializedGlobalModel>,
```

Comment:
did you also consider just the existing `RoundParameters` of the coordinator, and adding the global model on top? it's not a huge difference but it would avoid duplication...

also, is it necessary that they all have to be `Option`al?

Reply:
True! I can write:

```
pub struct RoundParametersData {
    /// The round parameters of the current round.
    pub round_parameters: Option<RoundParameters>,


    /// The global model of the previous round.
    pub global_model: Option<SerializedGlobalModel>,
}
```

I will change that👍